app/domain/authentication/authn_jwt/validate_status.rb

```diff
@@ -0,0 +1,77 @@
+module Authentication
+  module AuthnJwt
+
+    ValidateStatus = CommandClass.new(
+      dependencies: {
+        fetch_authenticator_secrets: Authentication::Util::FetchAuthenticatorSecrets.new,
+        discover_identity_provider: Authentication::OAuth::DiscoverIdentityProvider.new,
+        create_signing_key: Authentication::AuthnJwt::CreateSigningKeyInterface.new,
+        fetch_issuer_value: Authentication::AuthnJwt::FetchIssuerValue.new,
+        validate_uri_parameters: Authentication::AuthnJwt::ValidateUriBasedParameters.new,
+        validate_generic_status_validations: Authentication::AuthnJwt::ValidateGenericStatusValidations.new,
+        fetch_identity_from_token: Authentication::AuthnJwt::IdentityFromDecodedTokenProvider
+      },
+      inputs: %i[authenticator_status_input]
+    ) do
+
+      def call
+        validate_generic_status_validations
+        create_authentication_parameters
+        validate_service_id_exists
+        validate_uri_based_parameters
```

Comment:
what is verified under this validation class?

Reply:
In general, this validation class is responsible for validating the uri parameters of the request. at the moment it validates that the specified account exists and that the relevant webservice (created based on account, authenticator_name, and service_id) is whitelisted

Reply:
I think we should not use this command class because it purpose is to validate the uri parameters of the authenticate request not the status.